spec/unit/blockscore/connection_spec.rb

```diff
@@ -0,0 +1,52 @@
+module BlockScore
+  RSpec.describe Connection do
+
+    class ConnectionObject
+      include BlockScore::Connection
+
+      def resource # an unspoken contract.
+        nil
+      end
+    end
+
+    let(:connector) { ConnectionObject.new }
+
+    let(:api_key) { BlockScore.api_key }
+
+    before do
+      expect(HTTParty).to receive(:send).with(
+        :get,
+        "/foo?",
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.